python/glide-shared/glide_shared/config.py

```diff
@@ -107,23 +107,91 @@ def __init__(
         self.jitter_percent = jitter_percent
 
 
+class ServiceType(Enum):
+    """
+    Represents the types of AWS services that can be used for IAM authentication.
+    """
+
+    ELASTICACHE = 0
+    """Amazon ElastiCache service."""
+    MEMORYDB = 1
+    """Amazon MemoryDB service."""
```

Comment:
Nit. Don't need to specify values:
```suggestion
class ServiceType(Enum):
    """
    Represents the types of AWS services that can be used for IAM authentication.
    """

    ELASTICACHE
    """Amazon ElastiCache service."""
    MEMORYDB
    """Amazon MemoryDB service."""
```

Reply:
Code pattern in our codebase is define the numbers